packages/ove-app-maps/docs/MAP_LAYERS_JSON.md

```diff
@@ -0,0 +1,51 @@
+# MapLayers.json File
+
+The maps app can be configured by embedding the map layers configuration within the `config.json` file, or by providing a URL that points to a `MapLayers.json` file. The format of the JSON configuration remains the same in either of these two approaches.
+
+The `MapLayers.json` file contains an array of map layers. It has a structure similar to:
+
+```JSON
+[
+    {
+        "type": "ol.layer.Tile",
+        "visible": false,
+        "wms": false,
+        "source": {
+            "type": "ol.source.OSM",
+            "config": {
+                "crossOrigin": null,
+                "url": "https://{a-c}.tile.thunderforest.com/transport/{z}/{x}/{y}.png"
+            }
+        }
+    },
+    {
+        "type": "ol.layer.Vector",
+        "visible": false,
+        "wms": true,
+        "source": {
+            "type": "ol.source.Vector",
+            "config": {
+                "url": "/data/sampleGeo.json"
+            }
+        },
+        "style": {
+            "fill": {
+                "color": "#B29255"
+            },
+            "stroke": {
+                "color": "#715E3A",
+                "width": 4
+            }
+        },
+        "opacity": 0.7
+    }
+]
+```
+
+The app supports two types of map layers, `ol.layer.Tile` and `ol.layer.Vector`.
+
+Both types of map layers have a `visible` parameter that describes whether the layer is visible by default and a `wms` parameter which describes whether the layer is a displaying data from a [`Web Map Service`](https://www.opengeospatial.org/standards/wms). These parameters have boolean values.
```

Comment:
These parameters have ~~boolean~~ Boolean values.

Reply:
Thanks for remembering Boole was a person, :).